wgsl/index.bs

```diff
@@ -1184,16 +1217,14 @@ literal_or_ident
 
 <table class='data'>
   <thead>
-    <tr><th>Variable decoration keys<th>Valid values
+    <tr><th>Variable decoration keys<th>Valid values<th>Note
   </thead>
-  <tr><td>`binding`<td>u32 literal
-  <tr><td>`builtin`<td>a Builtin Decoration Identifier (listed below)
-  <tr><td>`location`<td>u32 literal
-  <tr><td>`set`<td>u32 literal
+  <tr><td>`binding`<td>non-negative i32 literal<td>See [[#resource-interface]]
```

Comment:
Doesn't SPIR-V allow specialization constants here? If so, can we add an issue for allowing that in WGSL some time in the future as well?

Reply:
No. Set and binding are literals in SPIR-V.